src/v/pandaproxy/schema_registry/api.h

```diff
@@ -59,6 +60,7 @@ class api {
     ss::sharded<schema_id_cache> _schema_id_cache;
     ss::sharded<pandaproxy::schema_registry::service> _service;
     ss::sharded<pandaproxy::schema_registry::seq_writer> _sequencer;
+    ss::sharded<ss::abort_source> _as;
```

Comment:
i don't think you need a sharded<abort_source> here, right? you can add a non-sharded abort source into `_service` and then in `service::stop` you stop the member abort source (ie. _as.stop()). then api::stop doesn't need any modification because it already invokes service::stop?